lxqt-session/src/procreaper.cpp

```diff
@@ -111,5 +111,5 @@ void ProcReaper::stop(const std::set<int64_t> & excludedPids)
         QMutexLocker guard{&mMutex};
         mShouldRun = false;
     }
-    QThread::wait(std::chrono::seconds(5));
+    QThread::wait(5000000); // 5 seconds
```

Comment:
5000?

Reply:
Fixed. Read the documentation of `QThread::usleep()`. It's enough for today.